tests/libkubevirt/config/kvconfig.go

```diff
@@ -91,9 +91,17 @@ func PatchWorkloadUpdateMethodAndRolloutStrategy(kvName string, virtClient kubec
 // UpdateKubeVirtConfigValueAndWait updates the given configuration in the kubevirt custom resource
 // and then waits  to allow the configuration events to be propagated to the consumers.
 func UpdateKubeVirtConfigValueAndWait(kvConfig v1.KubeVirtConfiguration) *v1.KubeVirt {
-	kv := testsuite.UpdateKubeVirtConfigValue(kvConfig)
-
-	waitForConfigToBePropagated(kv.ResourceVersion)
+	kv, changed := testsuite.UpdateKubeVirtConfigValue(kvConfig)
+	if changed {
```

Comment:
Why not `waitForConfigToBePropagated(kv.ResourceVersion)`?

Reply:
IMHO if I can request for a kubevirt configuration change, which controlled by virt-operator, the idea of virt-operator responding (somehow - > hoping it's not too complicated) that my change has been applied/synced is nicer then verifying the operands directly.

Reply:
Right but here the operator could be synced (observed version matches) but the components would not observe the new config. Meaning you could test different config that you think you do. This would only work if we would remove the `virt-config` (dynamic reconciliation of config by each component) and adapt a model where operator checks if components need to be redeployed based on config. But this is not a case as of now so this change would just introduce more undefined tests and flakes